src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.js

```diff
@@ -277,3 +277,72 @@ export function transformFormToAppointment(state) {
     schedulingMethod: 'direct',
   };
 }
+
+export function transformFormToVAOSCCRequest(state) {
```

Comment:
New transform to take our form data and put it into the format the VAOS services needs when submitting